### DISCUSSION TITLE
InvestmentFundETF

### DISCUSSION THREAD

**Discussion by samwant** [schemaorg/schemaorg#3416] [Ideas]
New type to accompany a quickly growing industry: 

**Gap in Existing Schema Types:**

While the existing [schema.org](http://schema.org/) types `FinancialProduct` and `InvestmentOrDeposit` provide a general framework for financial products and investments, they lack the specificity to represent Exchange Traded Funds (ETFs) effectively. This can lead to inaccurate or incomplete information being displayed in search results.

**Name:**

The new schema type will be named `InvestmentFundETF`. This name clearly identifies the type as an ETF and differentiates it from other `InvestmentFund` types.

**Properties:**

In addition to the properties inherited from `FinancialProduct` and `InvestmentOrDeposit`, the `InvestmentFundETF` type will include the following specific properties:

- `underlyingIndex`: The underlying index the ETF tracks (e.g., S&P 500).
- `assetClass`: The asset class of the underlying assets (e.g., equities, bonds, commodities).
- **`managementFee`**: The annual management fee of the ETF.
- **`netAssets`**: The net assets of the ETF (as of*)
- **`dividendFrequency`**: Schedule of distributions for the ETF (e.g., monthly, quarterly, yearly)

Thoughts?

**Comment by gmboxsmg2psn-collab** [comment]
This is an interesting discussion, especially because structured data only works well when information is kept consistent across every touchpoint. The same principle applies to customer communication: a [CommsHub messaging platform](https://commshub.com/) can help teams manage conversations across channels without losing context or sending conflicting updates.

**Comment by samwant** [comment]
Picking this back up — I've reframed it as a concrete proposal in #4891, now that InvestmentFund exists in Pending. It's scoped as an ExchangeTradedFund subtype rather than a standalone type. Continuing there rather than here.
